dashboard/test/controllers/lti_v1_controller_test.rb

```diff
@@ -50,45 +56,37 @@ def create_valid_jwt_raise_error
     create_jwt_and_stub(payload, true)
   end
 
-  test 'given no params, return unauthorized' do
+  test 'login - given no params, return unauthorized' do
     get '/lti/v1/login', params: {}
     assert_response :unauthorized
   end
 
-  test 'given a client_id that doesn not exist, return unauthorized' do
+  test 'login - given a client_id that doesn not exist, return unauthorized' do
     get '/lti/v1/login', params: {client_id: '', iss: @integration.issuer}
     assert_response :unauthorized
   end
 
-  test 'given a platform_id that does not exist, return unauthorized' do
+  test 'login - given a platform_id that does not exist, return unauthorized' do
     get '/lti/v1/login/wrong-id', params: {}
     assert_response :unauthorized
   end
 
-  test 'given a valid client_id via GET, return redirect' do
+  test 'login - given a valid client_id via GET, return redirect' do
     get '/lti/v1/login', params: {client_id: @integration.client_id, iss: @integration.issuer}
     assert_response :redirect
   end
 
-  test 'given a valid client_id via POST, return redirect' do
+  test 'login - given a valid client_id via POST, return redirect' do
     post '/lti/v1/login', params: {client_id: @integration.client_id, iss: @integration.issuer}
     assert_response :redirect
   end
 
-  test 'given a valid platform_id, return redirect' do
+  test 'login - given a valid platform_id, return redirect' do
     get "/lti/v1/login/#{@integration.platform_id}", params: {}
     assert_response :redirect
   end
 
-  test 'given valid parameters, set cookies' do
```

Comment:
followup - you could add a test that the 'write_cache' method was called w/ valid parameters

Reply:
Is there some cool method for asserting a method was invoked? I've dug around but can't find one.

Reply:
Yep! You'll want to mock or stub it, then use [Mock#expects](https://mocha.jamesmead.org/Mocha/Mock.html#expects-instance_method)